frontend/src/app/minigames/OddOnesOut.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+    // Note: I will remove the uneccessary comments/imports when I am done! 
+    // eslint-disable-next-line
+    Tooltip, Whisper, Alert, Dropdown, FlexboxGrid, Icon, IconButton
+} from "rsuite";
+
+// Primarily pulled from app header
+
+class OddOnesOut extends React.Component {

```

Comment:
Very small and nitpicky comment, but I think OddOneOut (in the singular) sounds better.